sig-service-catalog/annual-report-2020.md

```diff
@@ -0,0 +1,114 @@
+# Kubernetes SIG Service Catalog - 2020 Annual report
+Konstantin Semenov, Jonathan Berkhahn
+
+
+[Source](https://github.com/kubernetes/community/blob/master/committee-steering/governance/annual-reports.md)
+
+## Checklist
+- [x] Read about the process [here](https://git.k8s.io/community/committee-steering/governance/annual-reports.md#reporting-process)
+- [ ] Copy this template into a new document and share with your mailing list/slack channel/meeting on whatever platform (gdocs, hackmd, etc.) that the team prefers.
+- [x] Remove sections that are not applicable (example: if you are a working group, delete the special interest group questions)
+- [ ] Pick graphs from Devstats to pull supporting data for your responses.
+- [ ] Schedule a time with your Steering liaison and other Chairs, TLs, and Organizers of your group to check-in on your
+  roles as Chair or Working Group Organizer.
+  If anyone would rather meet 1:1, please have them reach out to the liaison directly, we are happy to.
+  We’d like to talk about: challenges, wins, things you didn’t know before but wish you did, want to continue in the
+  role or help finding a replacement; and lastly any feedback you have for us as a body and how we can help you
+  succeed and feel comfortable in these leadership roles.
+- [x] PR this document into your community group directory in kubernetes/community (example: sig-architecture/)
+    - [x] by March 8th, 2021
+    - [x] titled: annual-report-YEAR.md
+- [x] are there any responses that you’d like to share privately first? steering-private@kubernetes.io or tag your liaison in for discussion.
+
+## Operational
+1. How are you doing with operational tasks in SIG-governance.md?
+    1. Is your README accurate? have a CONTRIBUTING.md file?
+
+       Yes, the [README](https://github.com/kubernetes/community/blob/master/sig-service-catalog/README.md) is accurate.
+    2. All subprojects correctly mapped and listed in sigs.yaml?
+
+       Yes, our [subprojects](https://github.com/kubernetes/community/blob/master/sig-service-catalog/README.md#subprojects) are current.
+    3. What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date?
+
+       Are you keeping recordings up to date/trends in community members watching recordings?
+
+       We have a low profile bi-weekly meeting, with [notes and agenda up to date](https://docs.google.com/document/d/17xlpkoEbPR5M6P5VDzNx17q6-IPFxKyebEekCGYiIKM/edit#).
+       [Meeting recordings](https://www.youtube.com/watch?v=ukPj1sFFkr0&list=PL69nYSiGNLP2k9ZXx9E1MvRSotFDoHUWs) have not been uploaded since late 2019.
```

Comment:
😬 Do you know what the issue is here? Uploading them is required for transparency reasons. It also helps those that might be interested in your SIG get the latest info on it.